nl/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
+WantToCaptureForm? = Wilt u de waardes die u heeft ingevuld onthouden?
+WantToSavePasswordObscured? = Wachtwoord Manager kan deze inloggevens onthouden en automatisch invullen de volgende keer dat u terugkomt op deze website.##Wilt u dat de Wachtwoord Manager deze inloggevens onthoudt?
+WantToSavePasswordEncrypted? = Wachtwoord Manager kan deze inloggevens onthouden en automatisch invullen de volgende keer dat u terugkomt op deze website.##Wilt u dat de Wachtwoord Manager deze inloggevens onthoudt?
+xWantToSavePasswordObscured? = Wilt u deze gebruikersnaam en dit wachtwoord onthouden?##Opgeslagen waarden zijn niet beschermd door een wachtwoord.
+xWantToSavePasswordEncrypted? = Wilt u uw gebruikersnaam en wachtwoord onthouden?##Opgeslagen waarden zijn beschermd door een wachtwoord.
 #translator's note: # in following line will be converted to linefeed by program
-firstPassword = A master password will be used to protect sensitive single-signon information stored on your hard drive.  Future releases will strongly encrypt this data for maximum protection.##Select a master password.#(Leave fields blank if you don't want to use a master password).#
-password = Enter your master password.
+firstPassword = Een master wachtwoord wordt gebruikt om gevoelige informatie met eenmalige aanmelding te beschermen die is opgeslagen op uw hard drive.  Toekomstige releases zullen deze gegevens sterk vergrendelen voor maximale bescherming.##Selecteer een master wachtwoord.#(Laat de velden leeg als u geen master wachtwoord wilt gebruiken).#
+password = Voer uw master wachtwoord in.
 #translator's note: # in following line will be converted to linefeed by program
-newPassword = Select a new master password.#(Leave fields blank if you don't want to use a master password).#
-enterPassword = Enter a master password
-confirmPassword = Retype the master password
-confirmFailed_TryAgain? = The two master passwords were not identical.  Please enter the same master password in both fields.
-SelectUser = Select a username to be entered on this form.
-SelectUserWhosePasswordIsBeingChanged = Select the user whose password is being changed.
-PasswordNotification = For your convenience, the browser can remember your user names and passwords so you won't have to re-type them in the future.  Your passwords will be obscured before being saved on your hard drive.  Do you want this feature enabled?
-WalletNotification = You can save information that you enter on forms and later automatically prefill that information on other forms.  To save such information, select Edit/Save Form Info from the menu while viewing the form.
-noPrefills = There are no fields that can be prefilled.
-Never = Never for this site
-Confirm = Confirm
-PromptForPassword = Password
+newPassword = Selecteer een nieuw master wachtwoord.#(Laat de velden leeg als u geen master wachtwoord wilt gebruiken).#
+enterPassword = Voer een master wachtwoord in
+confirmPassword = Typ het master wachtwoord opnieuw
```

Comment:
CORRECTION NEEDED, it should be: ''Typ het master wachtwoord opnieuw in''.

- ''in'' added.